src/com/palmergames/bukkit/towny/object/Nation.java

```diff
@@ -171,20 +168,30 @@ public boolean hasTown(Town town) {
 		return towns.contains(town);
 	}
 
-	public void addTown(Town town) throws AlreadyRegisteredException {
+	public void addTown(Town town) {
+		towns.add(town);
+	}
 
-		if (hasTown(town))
-			throw new AlreadyRegisteredException();
-		else if (town.hasNation())
-			throw new AlreadyRegisteredException();
-		else {
-			towns.add(town);
-			town.setNation(this);
-			
-			BukkitTools.getPluginManager().callEvent(new NationAddTownEvent(town, this));
+	/**
+	 * Only to be called from the loading methods.
+	 * 
+	 * @param capital - Town to make capital.
+	 * @throws EmptyNationException Thrown when no capital can be set.
+	 */
+	public void forceSetCapital(Town capital) throws EmptyNationException {
+
+		if (towns.isEmpty())
+			throw new EmptyNationException(this);
+		
+		try {
+			if (capital.getNation().equals(this))
+				setCapital(capital);
```

Comment:
Doesn't this need a return statement since otherwise it will continue to `findNewCapital`?

Reply:
Good catch.